smartsim/entity/model.py

```diff
@@ -428,40 +428,34 @@ def _stringify_id(_id: int) -> str:
 
             raise TypeError(f"Argument is of type '{type(_id)}' not 'int'")
 
-        _invalid_input_message = (
-            "Expected a cpu pinning specification of type iterable of ints or "
-            f"iterables of ints. Instead got type `{type(pin_ids)}`"
-        )
+        try:
```

Comment:
While you're in here, could you tweak/fix the docstring? My inner pedant would be stoked if you swap `Create a comma-separated string CPU ids` with `Create a comma-separated string of CPU ids` and wrap None w/backticks `None`

Reply:
Totally reasonable! Will add!!